on the dangers of using unprotected Evidence and the resulting responsibilites (#332)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

The nature of how the Verifier manages to validate the signatures produced by the Attester is critical to the secure operation of a remote attestation system, but is not the subject of standardization within this architecture.

A conveyance protocol that provides authentication and integrity protection can be used

> to convey Evidence that is otherwise unprotected (e.g., not signed). Appropriate conveyance of unprotected Evidence (e.g., {{-uccs}} via a Secure Channel) relies on the following conveyance protocol's protection capabilities:

Reviewer: ```suggestion
to convey Evidence that is otherwise unprotected (e.g., not signed). Appropriate conveyance of unprotected Evidence (e.g., {{-uccs}} via a secure channel) relies on the following conveyance protocol's protection capabilities:
```

> to convey Evidence that is otherwise unprotected (e.g., not signed). Appropriate conveyance of unprotected Evidence (e.g., {{-uccs}} via a secure channel) relies on the following conveyance protocol's protection capabilities:

Reviewer: ```suggestion
to convey Evidence that is otherwise unprotected (e.g., not signed). Appropriate conveyance of unprotected Evidence (e.g., {{-uccs}}) relies on the following conveyance protocol's protection capabilities:
```

   1. The key material used to authenticate and integrity protect the conveyance channel is trusted by the Verifier to speak for the Attesting Environment(s) that collected Claims about the Target Environment(s).
   2. All unprotected Evidence that is conveyed is supplied exclusively by the Attesting Environment that has the key material that protects the conveyance channel
   3. The root of trust protects both the conveyance channel key material and the Attesting Environment with equivalent strength protections.

As illustrated in {{-uccs}}, an entity that receives unprotected Evidence via a trusted conveyance channel always takes on the responsibility of vouching for the Evidence's authenticity and freshness.
If protected Evidence is generated, the Attester's Attesting Environments take on that responsibility.

> In cases where unprotected Evidence is processed by a Verifier, Relying Parties have to trust the Verifier's capabilities of handling Evidence in a manner that renders the unprotected Evidence's authenticity and freshness believable at the time of Attestation Result generation.

Reviewer: ```suggestion
In cases where unprotected Evidence is processed by a Verifier, Relying Parties have to trust the Verifier is capable of handling Evidence in a manner that preserves the Evidence's authenticity and freshness.
```
